modules/stringtie/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to the `stringtie` module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0] - 2021-11-30
+
+This release was authored by Krysta Coyle.
+
+- awk script necessary to introduce XS tags into STAR-aligned BAMs. Without XS tags, stringtie generates single-exon transcripts.
+- GTF mandatory, highly recommended for well-annotated genomes.
```

Comment:
I'm not sure what "GTF mandatory, highly recommended" means... Is this related to the output format? Is the output format modifiable?

Reply:
I see how this was confusing. Updated to specify 'reference GTF'.